hack/wicd-unit.sh

```diff
@@ -18,7 +16,10 @@ if [ -z "$KUBE_SSH_KEY_PATH" ]; then
     return 1
 fi
 
-# Ensure the image used in the release job has ssh installed, as this will be used to remotely run tests against a
-# Windows VM. Running a dummy command to ensure connection to the VM is possible
-ssh -o StrictHostKeyChecking=no -i $KUBE_SSH_KEY_PATH $INSTANCE_USERNAME@$INSTANCE_ADDRESS "dir"
+# Build unit tests for Windows. Specifically targetting packages used by WICD.
+GOOS=windows GOFLAGS=-v go test -c ./pkg/winsvc/... -o wicd_unit.exe
+
+# Run the unit tests against the Windows host
+scp -o StrictHostKeyChecking=no -i $KUBE_SSH_KEY_PATH wicd_unit.exe $INSTANCE_USERNAME@$INSTANCE_ADDRESS:wicd_unit.exe
+ssh -o StrictHostKeyChecking=no -i $KUBE_SSH_KEY_PATH $INSTANCE_USERNAME@$INSTANCE_ADDRESS "wicd_unit.exe"
```

Comment:
Does it matter the working directory in the Windows host where it will be invoked? I assume will default to the user profile.

Reply:
It shouldn't matter at all. Since this is for CI purposes the home directory was used for simplicity.